bgpd/bgp_updgrp_adv.c

```diff
@@ -953,6 +986,8 @@ void subgroup_default_originate(struct update_subgroup *subgrp, bool withdraw)
 	dest = bgp_safi_node_lookup(bgp->rib[afi][safi_rib], safi_rib, &p,
 				    NULL);
 
+	// TODO BMP hook call for rib-out pre-policy
```

Comment:
i do not support the default originate atm so I left this TODO in case someone else has time to implement it

Reply:
maybe change the comment so it's related to default originate rather than rib-out pre-policy? would it be easier for someone to find if they want to support this in the future?

Reply:
sure, will do!